front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -29,6 +30,8 @@ type ManchetteWithSpaceTimeChartProps = {
   waypointsPanelData?: WaypointsPanelData;
   conflicts?: Conflict[];
   workSchedules?: PostWorkSchedulesProjectPathApiResponse;
+  totalTrains: number;
+  allTrainsProjected: boolean;
```

Comment:
As they both work together and are only here for the projection loader message, should we put them in an object to have less props to pass across all components ?

Reply:
Yes we can, but how would you name this object ?

Reply:
`ProjectionLoaderData` ? Like we do for other part of the simulation